doc/index.rst

```diff
@@ -20,204 +21,59 @@ Overview
-
-.. toctree::
-    :maxdepth: 1
-
-    using_xgboost
-
-.. toctree::
-    :maxdepth: 2
-
-    xgboost
-
-
-************
-Scikit-Learn
-************
-A managed enrionment for Scikit-learn training and hosting on Amazon SageMaker
-
-.. toctree::
-    :maxdepth: 1
 
-    using_sklearn
+The SageMaker Python SDK supports managed training and inference for a variety of machine learning frameworks:
 
 .. toctree::
     :maxdepth: 2
 
-    sagemaker.sklearn
-
-*******
-PyTorch
-*******
-A managed environment for PyTorch training and hosting on Amazon SageMaker
-
-.. toctree::
-    :maxdepth: 1
+    frameworks/index
 
-    using_pytorch
-
-.. toctree::
-    :maxdepth: 2
-
-    sagemaker.pytorch
-
-*******
-Chainer
-*******
-A managed environment for Chainer training and hosting on Amazon SageMaker
-
-.. toctree::
-    :maxdepth: 1
-
-    using_chainer
-
-.. toctree::
-    :maxdepth: 2
-
-    sagemaker.chainer
-
-**********************
-Reinforcement Learning
-**********************
-A managed environment for Reinforcement Learning training and hosting on Amazon SageMaker
-
-.. toctree::
-    :maxdepth: 1
-
-    using_rl
-
-.. toctree::
-    :maxdepth: 2
-
-    sagemaker.rl
-
-***************
-SparkML Serving
-***************
-A managed environment for SparkML hosting on Amazon SageMaker
-
-.. toctree::
-    :maxdepth: 2
-
-    sagemaker.sparkml
 
 ********************************
 SageMaker First-Party Algorithms
 ********************************
-Amazon provides implementations of some common machine learning algortithms optimized for GPU architecture and massive datasets.
+Amazon SageMaker provides implementations of some common machine learning algorithms optimized for GPU architecture and massive datasets.
 
 .. toctree::
     :maxdepth: 2
 
-    sagemaker.amazon.amazon_estimator
-    factorization_machines
-    ipinsights
-    kmeans
-    knn
-    lda
-    linear_learner
-    ntm
-    object2vec
-    pca
-    randomcutforest
-
-*****************************************
-Amazon SageMaker Operators for Kubernetes
-*****************************************
-Amazon SageMaker Operators for use with Kubernetes.
-
-.. toctree::
-    :maxdepth: 2
+    algorithms/index
 
-    amazon_sagemaker_operators_for_kubernetes
 
-*********
-Workflows
-*********
-SageMaker APIs to export configurations for creating and managing Airflow workflows.
+*************
+Orchestration
+*************
+Orchestrate your SageMaker training and inference jobs with Kubernetes and Airflow.
 
 .. toctree::
     :maxdepth: 1
 
-    using_workflow
+    kubernetes/index
+    airflow/index
+
+
+*************************
+Amazon SageMaker Debugger
+*************************
+You can use Amazon SageMaker Debugger to automatically detect anomalies while training your machine learning models.
 
 .. toctree::
     :maxdepth: 2
 
-    sagemaker.workflow.airflow
+    amazon_sagemaker_debugger
+
 
 *********************************
 Amazon SageMaker Model Monitoring
```

Comment:
does it make sense to keep these last couple sections (model monitoring and processing) even though they've also been moved to the training/inference folders?